addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3510,12 +3716,22 @@ export class OdooEditor extends EventTarget {
                 this._handleSelectionInTable(ev);
             }
         }
+        if (!this._rowUi.classList.contains('o_open') && !this._columnUi.classList.contains('o_open')) {
+            const uiToToggle = this._resizingTable ? [false, false] : this._isHoveringTableFirstOuter(ev);
+            this._toggleTableUi(...uiToToggle);
```

Comment:
I suggest refactoring this to:
```js
if (this._resizingTable) {
    this._toggleTableUi(false, false);
} else {
    // inlining of _isHoveringTableFirstOuter
    this._toggleTableUi(row, column);
}
```

Reply:
The `_isHoveringTableFirstOuter` function is used only once and has a weird name _because_ it is niche, which is why I wanted to inline it in the first place.

Reply:
Done in separate commit.